package.json

```diff
@@ -6,14 +6,13 @@
   "scripts": {
     "current": "node -e \"console.log(require('./package.json').version)\"",
     "clean": "rm -rf dist",
-    "build": "webpack --config=webpack.config.js --mode=production",
+    "build": "tsc -m es6 --outDir dist",
```

Comment:
It seems like setting an `outDir` will cause people to have to include `dist` when importing. Is that correct? If so, is there a way around that?

Reply:
hmmm no this used to work because the package.json sets `"main": "dist/highlighter.js"`, but that won't work without the webpack bundle, i think i need to add an `index.ts` that exports everything and set the main to that.